content/en/docs/ops/configuration/multi-revision/gateways/index.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Configuring and managing gateways with multiple revisions
+description: Configuring and managing gateways with multiple revisions.
+weight: 25
+keywords: [kubernetes,upgrading]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+With a single `IstioOperator` CR, any gateways defined in the CR (including the `istio-ingressgateway` installed in the
+default profile) are upgraded in place, even when the canary control plane method is used. This is
+undesirable - gateways are a critical component that should be upgraded last, when the new version control and data
+planes are verified to be working.
+The recommended way to upgrade gateways is to define and manage them in a separate `IstioOperator` CR, separate from
+the one used to install and manage the control plane.
+
+{{< warning >}}
+To avoid problems with `.` (dot) not being a valid character in some Kubernetes paths, the revision name should not
```

Comment:
> {{< tip >}} In a production environment, a better revision name would correspond to the Istio version. However, you must replace . characters in the revision name, for example, revision=1-6-8 for Istio 1.6.8, because . is not a valid revision name character. {{< /tip >}}

I think this should be `tip` as mentioned in https://github.com/istio/istio.io/commit/4c987afffb7541e09e42eb00e3cc3cc90462c293 ?

Reply:
To me a tip implies that the other way still works, but this is a better way. The situation here is that the other way will not work, so imo a warning is more appropriate. I'll let @frankbu make the official docs call on this.